ConfigurePanel.qml

```diff
@@ -112,7 +112,14 @@ AlgDialog {
             id: launchPhotoshopCheckBox
 
             function reload() {
-              checked = alg.settings.value("launchPhotoshop", false);
+              var launchPhotoshop = alg.settings.value("launchPhotoshop", false);
```

Comment:
A common way in js to cast from string to bool is:
var mybool = !!mystring;
You can else simplify with checked = launchPhotoshop === "true" || launchPhotoshop ;